tempesta_fw/t/functional/helpers/control.py

```diff
@@ -30,6 +31,7 @@ def __init__(self, binary, uri=''):
         in `form_command()` function. This would allow to update options until
         client will be started. See `Wrk` class for example
         """
+        remote.connect_client()
```

Comment:
I'm not sure, that running test cases as standalone programs is required. Running only a few test cases instead of entire test framework is desired. Actually old variant have at lest one serious drawback: you just can't interrupt test cases gracefully, so Tempesta, backend, and client may be still running after interruption of the test cases. You have added a possibility to run only specific test cases via `run_tests.py`. I really like that. 

And concerning  re-importing all modules. As i said i'm not good in python, so there can be dirty python code and wrong python libraries usage. On some reason i don't have issues with `paramiko` you have faced, and i see only one call of `remote.create_node()` per node on current master. If you say the change is needed, then ok, let's have `connect()` and `disconnect()` present in `TestCase` classes. Is it possible to move  `connect()` and `disconnect()` to `unittest.Testcase.setUpClass()` and `tearDownClass()`?